webapp/components/PasswordReset/Request.vue

```diff
@@ -68,8 +69,11 @@ export default {
     }
   },
   computed: {
+    email() {
+      return normalizeEmail(this.formData.email)
+    },
     submitMessage() {
-      const { email } = this.formData
+      const { email } = this
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-05T17:09:24Z" title="Tuesday, November 5th 2019, 6:09:24 pm +01:00">Nov 5, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -68,8 +69,11 @@ export default {
     }
   },
   computed: {
+    email() {
+      return normalizeEmail(this.formData.email)
+    },
     submitMessage() {
-      const { email } = this.formData
+      const { email } = this
```


I think this is the graphql mutation which has Boolean as return type. Thus we cannot take the normalized email from the backend but have to normalize it client-wise.